src/modules/mount/main.py

```diff
@@ -321,6 +327,12 @@ def run():
         return (_("Configuration Error"),
                 _("No partitions are defined for <pre>{!s}</pre> to use.").format("mount"))
 
+    # swap
+    swap_partitions = [p['device'] for p in partitions if p['fs'] == 'linuxswap']
+    swap_device="".join(map(str,swap_partitions))
+    if ( swap_device != "" ):
+        enable_swap_partition(swap_device)
```

Comment:
Why not simply pass `swap_partitions` to this function instead and have it enable them?